src/coreclr/jit/compiler.h

```diff
@@ -6114,7 +6119,8 @@ class Compiler
     void optOptimizeBoolsGcStress(BasicBlock* condBlock);
 #endif
 public:
-    void optOptimizeLayout(); // Optimize the BasicBlock layout of the method
+    PhaseStatus optInvertLoops();    // Invert while loops so they're entered at top and tested at bottom.
```

Comment:
```suggestion
    PhaseStatus optInvertLoops();    // Invert loops so they're entered at top and tested at bottom.
```
No need to use `while` here; structure of loops inverted will be described in function comments?